<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addWordPage</class>
 <widget class="QWidget" name="addWordPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>271</width>
     <height>312</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用户名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="username">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>经验值：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="expNum">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>等级：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rankNum">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>出题数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="passLevel">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>172</y>
     <width>321</width>
     <height>145</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>单词添加：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="answerEdit">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>20</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="submitButton">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>提交</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
